assets/style.styl

```diff
@@ -46,125 +49,136 @@ tour-tooltip
-    border-top 6px solid transparent
-    border-bottom 6px solid transparent
-    border-left 6px solid #fff
-    border-right none
-    z-index 1
-
-  .tether-target-attached-left &:after
-    border-top 7px solid transparent
-    border-bottom 7px solid transparent
-    border-left 7px solid #ddd
-    border-right none
-
-  // Right
-  .tether-target-attached-right &:before
-  .tether-target-attached-right &:after
-    right 100%
-    top 50%
-    transform translateY(-50%)
-
-  .tether-target-attached-right &:before
-    border-top 6px solid transparent
-    border-bottom 6px solid transparent
-    border-right 6px solid #fff
-    border-left 0
-    z-index 1
-
-  .tether-target-attached-right &:after
-    border-top 7px solid transparent
-    border-bottom 7px solid transparent
-    border-right 7px solid #ddd
-    border-left 0
-
-  // Top
-  .tether-target-attached-top &:before
-  .tether-target-attached-top &:after
-    top 100%
-    left 50%
-    transform translateX(-50%)
-
-  .tether-target-attached-top &:before
-    border-left 6px solid transparent
-    border-right 6px solid transparent
-    border-top 6px solid #fff
-    border-bottom none
-    z-index 1
-
-  .tether-target-attached-top &:after
-    border-left 7px solid transparent
-    border-right 7px solid transparent
-    border-top 7px solid #ddd
-    border-bottom none
-
-  // Bottom
-  .tether-target-attached-bottom &:before
-  .tether-target-attached-bottom &:after
-    bottom 100%
-    left 50%
-    transform translateX(-50%)
-
-  .tether-target-attached-bottom &:before
-    border-left 6px solid transparent
-    border-right 6px solid transparent
-    border-bottom 6px solid #fff
-    border-top none
-    z-index 1
-
-  .tether-target-attached-bottom &:after
-    border-left 7px solid transparent
-    border-right 7px solid transparent
-    border-bottom 7px solid #ddd
-    border-top none
-
-  // Diagonals
-  // Top/Bottom Left
-  .tether-target-attached-top.tether-target-attached-left &:before
-  .tether-target-attached-top.tether-target-attached-left &:after
-  .tether-target-attached-bottom.tether-target-attached-left &:before
-  .tether-target-attached-bottom.tether-target-attached-left &:after
-    left auto
-    right 10px
-    transform none
-
-  // Top/Bottom Right
-  .tether-target-attached-top.tether-target-attached-right &:before
-  .tether-target-attached-top.tether-target-attached-right &:after
-  .tether-target-attached-bottom.tether-target-attached-right &:before
-  .tether-target-attached-bottom.tether-target-attached-right &:after
-    left 10px
-    transform none
-
-  // Bottom offset
-  .tether-target-attached-bottom.tether-target-attached-left &:before
-  .tether-target-attached-bottom.tether-target-attached-left &:after
-  .tether-target-attached-bottom.tether-target-attached-right &:before
-  .tether-target-attached-bottom.tether-target-attached-right &:after
-    top auto
-
-  p
-    margin-bottom 15px
+  tour-tooltip-arrow
```

Comment:
Rewrote the arrow to an element so we can `ref` it in React. Working with pseudos would be harder.